styles/style.css

```diff
@@ -0,0 +1,150 @@
+h1, h2, h3, #welcome sub {
+  font-family: 'Major Mono Display', monospace;
+}
+
+nav, #welcome a, section:not(.welcome) {
+  font-family: 'Glegoo', serif;
+}
+
+nav ul {
+  display: grid;
+  grid-template: 100% / 60% 20% 20% ;
+  list-style-type: none;
+  padding: 20px;
+}
+
+h1 {
+  font-size: 50px;
+}
+
+#welcome {
+  height: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#welcome sub {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-bottom: 30px;
+  margin-top: 10px;
+  font-size: 30px;
+}
+
+#welcome a {
+  border: solid lightgray 1.5px;
+  border-radius: 10px;
+  text-decoration: none;
+  padding: 13px;
+  color: black;
+  margin: 25px;
+}
+
+#welcome a:hover {
+  background-color: #ecd9c6;
+  font-style: italic;
+}
+
+.quote {
+  height: 200px;
+  display: grid;
+  grid-template: 2fr 1fr / 100%;
+  border-top: solid lightgray 1.5px;
+  border-bottom: solid lightgray 1.5px;
+  font-style: italic;
+  padding: 50px;
+  font-family: 'Frank Ruhl Libre', serif;
+  background-color: #8DADCC;
+}
+
+.quote blockquote {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.quote p {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 40px;
+  align-items: flex-start;
+}
+
+#why-startrly {
+  display: grid;
+  grid-row: 1fr 2fr;
+}
+
+#why-startrly h2 {
+  text-align: center;
+  margin-top: 30px;
+}
+
+#why-startrly ul {
+  list-style-type: none;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 30px;
+}
+
+#why-startrly img {
+  margin-right: 10px;
+  float: left;
+}
+
+#startrly-team h2 {
+  text-align: center;
+  margin: 20px;
+}
+
+#startrly-team ul {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-around;
+  list-style: none;
+}
+
+#startrly-team img {
+  border-radius: 50%;
+  border: .5px lightgray solid;
+  padding: 3px;
+}
+
+#startrly-team li span {
+  text-align: center;
+  display: block;
+  margin: 5px;
+}
+
+#sponsors h2 {
+  text-align: center;
+  margin: 10px;
+}
+
+#sponsors ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  justify-content: space-around;
+}
```

Comment:
The sponsor section is currently aligned top, if you change it to center then it would look closer to the wireframe :)